tests/check_check_sub.c

```diff
@@ -553,6 +555,7 @@ START_TEST(test_ck_assert_str_le)
   record_test_name(tcase_name());
   ck_assert_str_le(s, t);
   ck_assert_str_le(s, s);
+  ck_assert_str_le(NULL, NULL);
```

Comment:
Oh, I see, this is resolved in a follow-on commit. Could the commits be combined?